pkg/operator/starter.go

```diff
@@ -198,12 +210,35 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		controllerContext.EventRecorder,
 	)
 
+	metricsObserver, err := observer.NewPrometheusMetricObserver("EtcdMetricsObserver",
+		[]observer.QueryHandler{
+			{
+				Name:  "EtcdFSyncDurationSeconds",
+				Query: "histogram_quantile(0.99, rate(etcd_disk_wal_fsync_duration_seconds_bucket[5m])",
+				Handler: func(ctx context.Context, recorder events.Recorder, currentValue string, valueType string) error {
```

Comment:
I'll find the other PR, but I expected to have one handler passed the current values for all metrics.

Reply:
can you elaborate? how you want to pass all metrics? I think we want to treat individual metrics separately... also we might watch alerts instead of using queries, dunno which one we would prefer for this one.

Reply:
i changed the handle so you have now full access to prometheus client (so you can do query/alert/etc.).